insights-notification-schemas-java/src/main/resources/schemas/Action.json

```
@@ -0,0 +1,123 @@
+{
+  "type": "object",
+  "title": "Notification message",
+  "comment": "Generated with https://hellosean1025.github.io/json-schema-visual-editor/",
+  "properties": {
+    "version": {
+      "type": "string",
+      "title": "Version of the notification schema",
+      "default": "2.0.0"
+    },
+    "bundle": {
+      "type": "string",
+      "title": "Bundle's name",
```

We could get git of all the `'s` in this file 😃
```suggestion
      "title": "Bundle name",
```